lib/ood_core/job/adapters/linux_host/templates/script_wrapper.erb.sh

```diff
@@ -1,5 +1,19 @@
 #!/bin/bash
-hostname
+SSH_HOSTS=(<%= ssh_hosts.join(' ').to_s %>)
+hostnames=`hostname -A`
+for host in $SSH_HOSTS
```

Comment:
`$SSH_HOSTS` needs to be `${SSH_HOSTS[@]}` to loop through all the elements in the array. As is it's just the first.

Reply:
Both $SSH_HOSTS AND ${SSH_HOSTS[@]} work as intended

Reply:
I'm not sure I follow what you mean by "work as intended". I meant it needs changed this `for element in array` statement.

See the difference these one liners
` ARR=(one two three); for ele in $ARR; do echo $ele; done`
`ARR=(one two three); for ele in ${ARR[@]}; do echo $ele; done`

You'll see the first only loops through the first element, while the second loops through all three, because the second expands the array.

Reply:
```#!bin/bash

SSHHOSTS='foo bar gee'
for host in $SSHHOSTS
do
        echo $host
done

for host in ${SSHHOSTS[@]}
do
        echo $host
done
```
The difference is my array is a string with multiple values

Reply:
the erb statement ssh_hosts.join(' ').to_s will result in a string like 'foo bar gee'

Reply:
Oh I see. What gets written though is `SSH_HOSTS=(foo bar gee)` which is a bash array.

One of the two has to change, because it doesn't work as is.